files/materialized_view_for_lanes.sql

```diff
@@ -49,12 +52,16 @@ as
   WITH NO DATA;
 
 -- First create an index that allows work/genre lookup. It's unique and incorporates license_pool_id so that the materialized view can be refreshed CONCURRENTLY.
-create unique index mv_works_for_lanes_unique on mv_works_for_lanes (works_id, genre_id, license_pool_id);
+create unique index mv_works_for_lanes_unique on mv_works_for_lanes (works_id, genre_id, license_pool_id, list_id);
```

Comment:
I'm not sure if it matters, but it seems to me that list_id should go first. It's more likely that you'd be doing a restriction based on list_id than that you'd be doing one based on license_pool_id.